### ISSUE TITLE
Wrong task ending

### ISSUE THREAD

**Issue by ks-s-a** [sun-zheng-an/gulp-shell#59]
Do stuff:

``` javascript
gulp.task('client-gulp-build', shell.task(['bower install', 'gulp --harmony build'], {
  cwd: './node_modules/project-client',
}));
```

In project-client task:

``` javascript
gulp.task('build', function(callback) {
  return runSequence(
    'clean',
    'client-init',
    ['build-widget', 'build-images', 'build-custom'],
    callback);
});
```

In 0.4.1-0.4.3 version of gulp-shell 'client-gulp-build' task executes all commands, but gulp-shell task doesn't complete itself. Next task doesn't start.

In 0.4.0 version all works correctly.

**Comment by pokono** [comment]
+1

**Comment by sun-zheng-an** [comment]
@ks-s-a Could you give me a simplified `gulpfile` that can reproduce this issue?

**Comment by agentschmitt** [comment]
I have the same issue in webstorm. the process of shell tasks is never finished.
var shell = require('gulp-shell');
gulp.task('test', shell.task(['echo test']));

**Comment by Dygerati** [comment]
+1

**Comment by sun-zheng-an** [comment]
@agentschmitt Thanks for the feedback.

Could you tell me the version of node and the platform you used?

**Comment by lplotni** [comment]
I run into the same problem on SnapCI as soon as I changed the pipeline configuration to use node 4.1.x.  The task based on gulp-shell doesn't terminate. If I run it locally (osx) with the same node version, I cannot reproduce it.

**Comment by sun-zheng-an** [comment]
Can someone write some failing tests for me? I cannot reproduce it in my local machine (Node 4.1.1 and OS X 10.11) or by [Travis CI](https://travis-ci.org/sun-zheng-an/gulp-shell).

**Comment by sun-zheng-an** [comment]
I just released `0.5.0`, it should fix all the issues like this one (running non-interactive commands and doesn't terminate properly at the end).

Please check it out.

**Comment by lplotni** [comment]
@sun-zheng-an works on SnapCi :+1:
